docs/index.bs

```diff
@@ -3175,6 +3222,52 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Return |response|.
   </section>
 
+  <section algorithm>
+    <h3 id="parse-urlpattern-algorithm"><dfn>ParseURLPattern</dfn></h3>
+      : Input
+      :: |rawPattern|, a {{RouterCondition/urlPattern}}"
```

Comment:
```suggestion
      :: |rawPattern|, a [=string=]
```

(The type is `[=string=]`; `{{RouterCondition/urlPattern}}` is a field name, not a type.)

Reply:
I may be misunderstanding, but is the type string here? I thought `URLPatternCompatible` is the correct type.

Reply:
Oh, this is a good catch!!

Reply:
Thanks for noticing this.  done.